src/views/appShell/AppLogIn/contexts/UserSessionContext.tsx

```diff
@@ -50,12 +45,8 @@ export function UserSessionProvider({ children }: IUserSessionContextProps) {
         return username !== '' && token !== '';
```

Comment:
Those lines are not used anymore on the app, so we can delete them

So the if-else statement can be deleted, because session.storage.getItem('isAuthenticadted) is not used

Reply:
This is still used 'src/views/appShell/AppLogIn/components/PrivateRoute.tsx'